tests/manage/pv_services/test_delete_project_with_pvc_and_pods.py

```diff
@@ -0,0 +1,171 @@
+"""
+The purpose of this test case is to delete at least 2 projects:
+One project with just pvcs and the other with pvcs attached to a pod
+"""
+import logging
+import time
+import pytest
+from itertools import cycle
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.resources.ocs import OCS
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-278")
+@pytest.mark.parametrize(
+    argnames=["pvcs_num"],
+    argvalues=[
+        pytest.param(
+            100, marks=pytest.mark.tier1
+        ),
+        pytest.param(
+            1000, marks=pytest.mark.scale
```

Comment:
With this marker, this test will be picked for tier1 and scale. Should these tests be separate ?

Reply:
@ebenahar your take on that?

Reply:
this looks fine. 1000 should be tested as part of scale suite and 100 as part of tier1. We are good

Reply:
Fine with that. Then this tier1 annotation should be removed from line 19. Otherwise, 1000 will be selected for both scale and tier1

Reply:
Understandable